test/Cover/DefaultPdfGeneratorTest.php

```diff
@@ -197,10 +203,7 @@ public function getMetadataFromConfig($optionKeys)
      */
     protected function getXetexPdfGenerator()
```

Comment:
Die Funktion ist eigentlich nicht mehr notwendig. Sie wäre sinnvoll, wenn sich die Tests alle selber ihr Objekt holen, um die Instanziierung nur an einer Stelle zu haben, aber das geht ja jetzt auch in setUp.